packages/localize/src/LocalizeMixin.js

```diff
@@ -62,7 +62,10 @@ export const LocalizeMixin = dedupeMixin(
         if (this.__localizeMessageSync) {
           return localize.msg(...args);
         }
-        return until(this.localizeNamespacesLoaded.then(() => localize.msg(...args)), nothing);
+        return until(
```

Comment:
Linting. Circle-ci build failed because of this. Not sure how it ended up in master (pre-commit hook should catch it), I've been trying to figure out why for a while now but without success. Could be that someone is using --no-verify flag but then still the circle-ci build should fail and not let it through..